back-end-side.UnitTests/Services/SortingServiceTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using back_end_side.DbFiles;
+using back_end_side.Services;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Record = back_end_side.Models.Record;
+
+namespace back_end_side.UnitTests.Services
+{
+    public class SortingServiceTests
+    {
+        [Fact]
+        public async Task ChangeCategory_OneRecordFromZaraExists_RecordFromZaraCategoryisChangedToEntertainment()
+        {
+            var context = await GetDatabaseContext();
+            var record1 = new Record
+            {
+                ID = 1,
+                Seller = "zara",
+                Purpose = "jeans",
+                Date = new DateTime(2022 - 11 - 19),
+                Amount = 20,
+                Category = "clothes",
+                IsCategorized = true
+            };
+            var record2 = new Record
+            {
+                ID = 1,
+                Seller = "zara",
+                Purpose = "jeans",
+                Date = new DateTime(2022 - 11 - 19),
+                Amount = 20,
+                Category = "entertainment",
+                IsCategorized = true
+            };
+            context.Add(record1);
+            var service = new SortingService(context);
+            service.ChangeCategory(record2, record2.ID);
+
+            Assert.Equal(record1.Category, record2.Category);
+
+        }
+        private async Task<ExpensesContext> GetDatabaseContext()
```

Comment:
Since this method is repeated it would be a good idea to move it to a separate file

Reply:
Excellent suggestion! It will be done in the near future.